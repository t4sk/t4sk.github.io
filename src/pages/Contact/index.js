import React from "react"
import PropTypes from "prop-types"
import Layout from "../../components/Layout"
import SEO from "../../components/SEO"
import "./index.css"
import translate from "../../translate"
import LANG from "./index.lang"

function Contact(props) {
  const { lang } = props

  const title = `${translate(
    LANG,
    lang,
    "Contact"
  )} | Smart Contract Programmer`
  return (
    <Layout lang={lang}>
      <SEO title={title} lang={lang} />
      <h3>{translate(LANG, lang, "Contact")}</h3>
      <div>smartcontractprogrammer@gmail.com</div>
    </Layout>
  )
}

Contact.propTypes = {
  lang: PropTypes.oneOf(["en", "jp"]).isRequired,
}

export default Contact
