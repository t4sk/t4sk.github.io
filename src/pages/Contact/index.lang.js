const LANG = {
  en: {
    Contact: "Contact",
  },
  jp: {
    Contact: "お問い合わせ",
  },
}

export default LANG
