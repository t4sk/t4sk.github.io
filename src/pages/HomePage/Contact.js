import React from "react"
import PropTypes from "prop-types"
import styles from "./Contact.module.css"

export function Contact(props) {
  return (
    <p>
      You can contact me at{" "}
      <span className={styles.email}>contact@smartcontractprogrammer.com</span>
    </p>
  )
}

Contact.propTypes = {}

export default Contact
