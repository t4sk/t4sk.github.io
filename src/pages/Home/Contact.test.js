import React from "react"
import { shallow } from "enzyme"
import { Contact } from "./Contact"

const props = {}

beforeEach(() => {})

test("it renders", () => {
  const component = shallow(<Contact {...props} />)

  expect(component).toMatchSnapshot()
})
