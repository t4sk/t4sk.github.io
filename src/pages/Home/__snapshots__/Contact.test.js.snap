// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<p>
  You can contact me at
   
  <span
    className="email"
  >
    contact@smartcontractprogrammer.com
  </span>
</p>
`;
